ClientApp/app/(tabs)/create/index.tsx

```diff
@@ -1,25 +1,534 @@
-import axiosInstance from '@/services/axiosInstance';
-import React from 'react';
-import { View, Text, TextInput, Button } from 'react-native';
-const handleSubmit = async () => {{
-  const response = await axiosInstance.get("user-service/user/673114e06f52646e079d1486");
-  console.log('============================================');
-  console.log(response);
-}
-
-}
+import React, { useEffect, useState } from 'react';
+import {
+  View,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  ScrollView,
+  StyleSheet,
+  Modal,
+  FlatList,
+  StatusBar,
+  Alert,
+} from 'react-native';
+import { useForm, Controller, useWatch } from 'react-hook-form';
+import DateTimePicker from '@react-native-community/datetimepicker';
+import { useRouter } from 'expo-router';
+import ConfirmButton from '@/components/ConfirmButton';
+import { IconPlacement } from '@/utils/constants/enums';
+import themeColors from '@/utils/constants/colors';
+import { hs, vs, mhs } from '@/utils/helpers/uiScaler';
+import { API_ENDPOINTS } from '@/utils/api/endpoints';
+import { createEvent } from '@/services/eventService';
+import { useSelector } from 'react-redux';
+
+const sports = ['Soccer', 'Basketball', 'Tennis', 'Swimming', 'Running', 'Football', 'Rugby'];
```

Comment:
I think there is an already created sports object somewhere in the app. You can maybe use that to avoid duplicate code and redundancy. I think @NicolasChelico used it for the Register profile page.

Reply:
import statement import supportedSports from "@/utils/constants/supportedSports";

Reply:
agreed! done.